content/en/mobile/_index.md

```diff
@@ -0,0 +1,81 @@
+
+The Datadog Oncall app enables on-call engineers to view alerts from Datadog on their mobile device. When notified to a Datadog alert by Pagerduty, Slack or other pager apps, you'll be able to investigate issues by opening monitor graphs and dashboards on your mobile device.
+
+## Installing
+
+The Datadog Oncall App is currently in beta, so it's not available generally in app stores. To download the app, use the specific invite links in these instructions.
+
+### iOS
+
+1. Install the [Testflight app][1] from the App Store.
+2. Open this invite link from your iPhone: [https://apple.co/2xEC5ke][2]. Allow the link to open in the Testflight app. This takes you to a page where you can download the Datadog app.
+3. Once the app downloads, open the app, select whether you are logging into the US or EU site, and log in with your Datadog credentials.
+
+### Android
+
+1. To install the android app, open this invite link from your android: [https://play.google.com/apps/testing/com.datadog.app][3]. This takes you to a page where you can download the Datadog app.
+2. Once the app downloads, open the app, select whether you are logging into the US or EU site, and log in with your Datadog credentials.
+
+### Logging in
+
+You can login using standard and Google authentication. [SAML login][4] is not currently supported. If you have SAML enabled but not strictly, you can access the mobile app by creating a new standard email account. An org admin needs to approve the newly created account.
+
+Switch organizations and log out from the **Account page**.
+
+## Monitors
+
+{{< img src="mobile/monitors.jpg" alt="monitor page" style="width:50%;">}}
+
+On the Monitors page, you can view and search all of the monitors that you have access to in your Datadog org. You can specify by field name and build specific search queries based on your tagging strategy. For more information about searching, see the [Manage Monitors Search section][5]. For example, to filter only metric monitor related to SRE team that are alerting, use the query `"status:Alert type:Metric team:sre"`. Click into individual alerts to see details, which can be filtered by type and by alert time. You can also mute the alert. Your ten most recent searches are saved so that you can faster access previous queries.
+
+**Note:** To set up or edit a monitor and alerts, you need set them up in the [Datadog web app][6]. All alerts set up in the app are enabled for the mobile app. For more information, see [Creating monitors][7].
+
+## Dashboards
+
+On the Dashboards page, you can view and search all of the dashboards that you have access to in your Datadog org. Click into an individual dashboard to view it.
+
+**Note:** To set up or edit a dashboard, you need to [login to the Datadog browser app][8]. For more information, see [Dashboards][9].
+
+## Account
+
+Switch organizations and log out from the Account page.
```

Comment:
Not understanding this segment which looks to be the same as above.

Reply:
resolved above. There is an account tab - you can see it really clearly from the app, and this is just describing what it does